test/format-test.cc

```diff
@@ -219,12 +219,12 @@ TEST(AllocatorTest, allocator_ref) {
   check_forwarding(alloc, ref3);
 }
 
-typedef allocator_ref<std::allocator<char>> TestAllocator;
+typedef allocator_ref<std::allocator<char>> StdAllocator;
```

Comment:
Let's follow the naming conventions while at it:

```c++
using std_allocator = allocator_ref<std::allocator<char>>;
```

Reply:
👍 Done